consensus/consensus_service.go

```diff
@@ -90,6 +90,11 @@ func (consensus *Consensus) UpdatePublicKeys(pubKeys []bls_cosi.PublicKeyWrapper
 		consensus.LeaderPubKey = &allKeys[0]
 		utils.Logger().Info().
 			Str("info", consensus.LeaderPubKey.Bytes.Hex()).Msg("My Leader")
+	} else {
+		utils.Logger().Error().
+			Msg("[UpdatePublicKeys] Participants is empty")
+		consensus.pubKeyLock.Unlock()
+		return 0
```

Comment:
why return early? If there is no participants. It doesn't mean it's an error. We don't prevent empty pubKeys anyway.

Reply:
okay.